src/endpoints.py

```diff
@@ -26,6 +27,28 @@
 app = Flask(__name__)
 assets = Environment(app)
 css = Bundle(const.PATH_CSS_INPUT, output=const.PATH_CSS_OUTPUT)
+socketio = SocketIO(app)
+app.config['SECRET_KEY'] = 'secret!'
```

Comment:
constants should be moved to the const file. Secrets however should not be pushed